using/src/activations.rs

```diff
@@ -0,0 +1,178 @@
+//! Methods to select which of a set of implementations to activate based on a schema's
+//! `Request`s.
+
+use crate::{Find, Found, Implementations, Request, Schema};
+
+/// An Activation references a request and contains the min and max versioned implementations
+/// available in the implementation registry which can satisfy that request, or None if no
+/// matching implementations were available.
+// type Activation<'schema, 'impls, T> = (&'schema Request, Find<'impls, T>);
+
+impl<'a> Schema<'a> {
```

Comment:
What is the reason for having this not part of `schema.rs`? It's definitely not standard, and from what I understand of this PR I don't see a reason for it to be separate, but maybe I am missing something!